src/EditorFeatures/CSharpTest/project.json

```diff
@@ -4,6 +4,10 @@
             "version": "14.3.25407",
             "suppressParent": "all"
         },
+        "RoslynDependencies.Microsoft.VisualStudio.Text.Internal": {
+           "version": "14.3.25407",
+           "suppressParent": "all"
+        },
```

Comment:
Surprising, but I accept that there are mysteries in the universe... :smile:

Perhaps we were embedding the interop types in the InteractiveWindow.